src/test/java/gov/healthit/chpl/aqa/pageObjects/AnnouncementsPage.java

```diff
@@ -45,5 +45,14 @@ public static WebElement savedAnnouncementInformatoin(final WebDriver driver) {
     public static WebElement announcementReportDetails(final WebDriver driver) {
         return driver.findElement(By.xpath("//*[@id=\"main-content\"]/div/ui-view/chpl-reports-announcements/div"));
     }
+
+    public static WebElement editExistingAnnouncement(final WebDriver driver) {
+        return driver.findElement(By.xpath("//tr[2]//td[6]//button[1]"));
```

Comment:
This looks like xpath to 2nd row and 6th column in announcement table. This won't work when there's no data in the table(like the announcement you assumed was added, did not add) or only one announcement in the table. If you are adding an announcement and then editing that specific announcement, and if it was to be in different row, you'd be editing a non intended announcement. Looping through table elements  and clicking 'Edit' button based on data in other columns (the announcement you intend to edit) is better way to go for it.